scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
Yeah, the docs seem to indicate that this doesn't just apply to a review app. if we deploy a branch with this app.json to production, I think it will run this script.